modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/util/DLValidatorImpl.java

```diff
@@ -93,6 +93,23 @@ public long getMaxAllowableSize(long groupId, String mimeType) {
 					groupId, mimeType)));
 	}
 
+	public Map<String, Long> getMimeTypeSizeLimit(long groupId) {
+		long companyId = _getCompanyId(groupId);
+
+		Map<String, Long> groupMimeTypeSizeLimit =
+			_dlSizeLimitManagedServiceFactory.getGroupMimeTypeSizeLimit(
+				groupId);
+		Map<String, Long> companyMimeTypeSizeLimit =
+			_dlSizeLimitManagedServiceFactory.getCompanyMimeTypeSizeLimit(
+				companyId);
+
+		companyMimeTypeSizeLimit.forEach(
+			(key, value) -> groupMimeTypeSizeLimit.merge(
+				key, value, (v1, v2) -> Math.min(v1, v2)));
```

Comment:
it's better to use the internal function `_min` instead `Math.min` because 0 is "not limit"

Reply:
And if you create a method on the DLValidatorImpl maybe the DLValidatorUtil method is a good idea :)

Reply:
I updated the source to use __min_ function following your comment 
About the move this function to _DLValidatorUtil_, I think we should keep it in DLValidatorImpl class because we can re-use __min_ function and the  _DLSizeLimitManagedServiceFactory_. Maybe I misunderstand your idea here, if that's true, could you guide me more
Thank you

Reply:
Oh,

No, sorry the misunderstanding
I was not telling to move to the util.
I was only asking if was worthy to create the method on the util, to let the use the method on the service when we cannot referente the service (as others utils generated by the build service action)

Normally on other services, the creation of the methods on the service/serviceUtil/serviceWrapper are generated automatically.
This is not the case, so I was asking if maybe (or not) the static method was interesting to create

sorry for the misunderstanding

Reply:
Hi @AliciaGarciaGarcia,

 Because the returned value of https://github.com/liferay-lima/liferay-portal/pull/3282/files/84daab021d77eeae050738030fd10a45d17d9c0b..bf87fbc27273619fbc09f2ed3a87fa33159ab8d7#diff-e794766240e1c2a0c1f555026169f7ab7414357579a92d011d82bf5b2cfbfe30R96 function depend on the configuration on each instance, so in my opinion, it should be implemented on [document/library/internal/util/DLValidatorImpl.java](https://github.com/liferay-lima/liferay-portal/pull/3282/files/84daab021d77eeae050738030fd10a45d17d9c0b..bf87fbc27273619fbc09f2ed3a87fa33159ab8d7#diff-e794766240e1c2a0c1f555026169f7ab7414357579a92d011d82bf5b2cfbfe30), and we will cannot get configuration information on DLValidatorUtil class. So we should keep the current implementation.

Reply:
👍🏻